common/Services/ComputeSystemService.cs

```diff
@@ -35,6 +49,17 @@ public async Task<List<ComputeSystemProviderDetails>> GetComputeSystemProvidersA
         {
             try
             {
+                // Work around for issue where the Dev Home's extension service uses classIds within a package to identify the extension, but doesn't handle
+                // multiple packages with the same extension classIds. We need to filter out the Dev Home extensions that are not within the current package.
+                // E.g the Hyper-V extension is in Dev Home Dev, Canary and preview builds, each with the same class Id.
+                // So Dev Home sees this as 3 separate extensions, causing us to query the same COM server up to 3 times depending on how many of the 3 are
+                // installed.
+                if (_devHomePackageFamilyName.Contains(extension.PackageFamilyName) &&

```

Comment:
Technically, package names are case-insensitive.  IIRC, we have bugs where we don't handle that correctly, but there is no reason to have another bug here. However, publisher ID is case sensitive. So, we should split it into two parts and compare separately or ignore publisher ID since we don't check if we trust the publisher here.

For reference see https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/package-identity-overview#is-package-identity-case-sensitive:~:text=Is%20package%20identity%20case,but%20compare%20case%2Dinsensitively

Reply:
I didn't think it mattered since it's a work around and will be removed when the core team fixes that issue, while also only applying to the Dev Home app specifically that we know would have the publisher with a capital. But I can change it, if you think it'd be an issue

Reply:
I don't think it's an issue here (at least not now). It's more along the lines "Microsoft is not following their own documentation/recommendations that tell how to compare package name".